src/common/containers/landing.js

```diff
@@ -59,41 +58,52 @@ class Landing extends Component {
             </div>
           </div>
         </div>
-
-        <section className={ `${styles.section} ${styles.sectionNoBorder} ${containerStyles.lightStrip}` }>
-          <div className={ `${styles.row} ${containerStyles.wrapper}` }>
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Scale to millions of installs</li>
-              <li>Available on all clouds and Linux OSes</li>
-              <li>No need for build infrastructure</li>
-            </ListDividedState>
-
-            <ListDividedState className={ styles.rowItemGrow }>
-              <li>Automatic updates for everyone</li>
-              <li>Roll back versions effortlessly</li>
-              <li>FREE for open source projects</li>
-            </ListDividedState>
+        <section className={ `${styles.section} ${styles.sectionTopBorderOnly}` }>
+          <div className={ `${containerStyles.wrapper}` }>
+            <HeadingTwo className={ styles.landingHeading }>
+              Publish your software for
+            </HeadingTwo>
+            <div className={ `${styles.row}` }>
+              <img className={ styles.brandLogo } src={images.debian} />
+              <img className={ styles.brandLogo } src="https://assets.ubuntu.com/v1/59190d38-suse_logo.svg" />
```

Comment:
I did wonder that. AFAIUI we should have all of the assets on the asset server.

Should I move the other images to the asset server?